pkgs/development/python-modules/ifcfg/default.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, fetchPypi
+, buildPythonApplication
+, python
+, iproute2
+}:
+
+buildPythonApplication rec{
+  pname = "ifcfg";
+  version = "0.23";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-YktaxUFpPM0+LOzDSMDogVTe1Kqt0lTLIxIS36r9vQI";
+  };
+
+  nativeBuildInputs = [
+    python
+  ];
+
+  propagatedBuildInputs = [
+    iproute2
+  ];
+
+  pythonImportCheck = [ "ipcfg" ];
```

The attribute name is wrong (preventing the check from running) along with the wrong module name.
```suggestion
  pythonImportsCheck = [ "ifcfg" ];
```

---

Good catch.  It was not obvious from the build output that it hadn't ran.